packages/react/src/patterns/blocks/ContentBlockSimple/ContentBlockSimple.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentItem } from '../../sub-patterns/ContentItem';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Block - Simple pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.copy Content block simple short copy to support the heading
+ * @param {string} props.linkType link type ( simple | jump | card )
+ * @param {object} props.link link object which includes url, link text and target properties.
+ * @param {string} props.heading Content block simple heading
+ * @param {object} props.cta cta object
+ * @param {object} props.contentGroup chilren content element
+ * @param {string} props.mediaType Media type, video or image
+ * @param {object} props.mediaData Data for renderimg the media
+ * @param {Array} props.items Array of objects with data for ContentItems
+ * @returns {*} Content block simple pattern
+ */
+const ContentBlockSimple = ({
+  copy,
+  heading,
+  items,
+  mediaType,
+  mediaData,
```

Comment:
I understand this was in the functional specs, but I don't know if we should include it right now. All it should be is an `Image` object because we're not ready to bring in video yet. I don't know. Just my thoughts.

Reply:
If that's the case then we should update the func specs to not even mention video being an option, since as far as I know (and have been doing) all the patterns accommodate `video` because it's listed in the specs